apps/api/src/shared/service/user-state.service.ts

```diff
@@ -20,8 +21,25 @@ export class UserStateService {
   async findByUser(userId: string): Promise<UserState> {
     const state: UserState = await this.userStateModel.findOne({ user: userId }).exec();
     if (state) {
-      state.logos = await this.userCompletedLogoService.findByState(state._id);
+      // state.logos = await this.userCompletedLogoService.findByState(state._id);
```

Comment:
should this be removed along the condition?

Reply:
yep, removed